frontend/public/components/utils/timestamp.tsx

```diff
@@ -13,11 +13,11 @@ const timestampFor = (mdate: Date, now: Date, omitSuffix: boolean) => {
     return '-';
   }
 
-  const timeAgo = now.getTime() - mdate.getTime();
+  const timeFromNow = Math.abs(now.getTime() - mdate.getTime());
   if (omitSuffix) {
     return dateTime.fromNow(mdate, undefined, {omitSuffix: true});
   }
-  if (timeAgo < 630000) { // 10.5 minutes
+  if (timeFromNow < 630000) { // 10.5 minutes
```

Comment:
I wouldn't assign `timeFromNow` to a variable to make sure we don't accidentally use it as the date we show. (With `Math.abs` it could be the wrong value.) We should add a comment explaining as well.

```suggestion
  // Show a relative time if within 10.5 minutes from the current time (in the past or future).
  if (Math.abs(now.getTime() - mdate.getTime()) < 630000) { // 10.5 minutes
```

Reply:
changed to `timeDifference`